client.go

```diff
@@ -69,3 +79,19 @@ const (
 // Job is a function that will execute like a cron job after a
 // certain amount of time to perform some kind of task.
 type Job func(Bot, Channel) error
+
+//BroadcastFilter returns true if message can be broadcasted to bot and Channel
+type CanBeBroadcasted func(botID string, channel Channel, msg Sendable) bool
```

Comment:
The function returns a bool, so the name seems to be more descriptive if is like "can-be", "is-whatever"...In the other hand... "filter" suffix seams to point to some kind of tool that excludes some parts of an input.

The idea of passing the message, is that this way the implementation of the filter can use the message to determine if it can be sent only considering his content (example: using a bad words dictionary)